core/help/help.py

```diff
@@ -123,6 +123,8 @@ def update_operators_text():
                     operator = operators.get(operator_text)
                     if type(operator) == str:
```

Comment:
Please use `isinstance` instead of `type`.

Reply:
Done